src/mono/wasm/debugger/BrowserDebugProxy/ValueTypeClass.cs

```diff
@@ -116,11 +117,11 @@ public async Task<JObject> ToJObject(MonoSDBHelper sdbAgent, bool forDebuggerDis
             string description = className;
             if (ShouldAutoInvokeToString(className) || IsEnum)
             {
-                int[] methodIds = await sdbAgent.GetMethodIdsByName(TypeId, "ToString", token);
-                if (methodIds == null)
-                    throw new InternalErrorException($"Cannot find method 'ToString' on typeId = {TypeId}");
-                var retMethod = await sdbAgent.InvokeMethod(Buffer, methodIds[0], token, "methodRet");
-                description = retMethod["value"]?["value"].Value<string>();
+                var toString = await sdbAgent.InvokeToStringAsync(new List<int>(){TypeId}, true, IsEnum, Id.Value, IsEnum ? BindingFlags.Default : BindingFlags.DeclaredOnly, token);
```

Comment:
```suggestion
                var toString = await sdbAgent.InvokeToStringAsync(new int[]{ TypeId }, true, IsEnum, Id.Value, IsEnum ? BindingFlags.Default : BindingFlags.DeclaredOnly, token);
```

.. and named arguments, because of the two bools.